app/views/teaching_events/show/_train-to-teach-event-summary.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="about-ttt-event">
+  <p>Watch presentations, ask questions, speak to advisers one-to-one.</p>
+</div>
+
+<figure>
+  <blockquote>
+    Absolutely loved this day. Very informative, inspiring and I'm
+    excited to change career. Thank you.
+  </blockquote>
+  <figcaption>Sarah, trainee teacher, London</figcaption>
```

Comment:
Can you delete the caption - we don't have names to use, so just going with the quotes